### ISSUE TITLE
Wiki linked dates will not parse in graph

### ISSUE THREAD

**Issue by DaveHenning** [pyrochlore/obsidian-tracker#26]
Tracker graphs only parse when using non wiki-linked dates.

Example table:

| Date           | Data Usage |     | Cap  |     |
| -------------- | ----------:| --- | ---- | --- |
| [[2021.03.01]] |       0.04 | MB  | 2.05 | GB  |
| [[2021.03.03]] |       0.04 | MB  | 2.05 | GB  |
| [[2021.03.17]] |       9.15 | MB  | 2.05 | GB  |
| [[2021.03.19]] |       9.15 | MB  | 2.56 | GB  |
| [[2021.03.24]] |      98.88 | MB  | 2.50 | GB  |

Leaving the wiki links on the dates does not allow the graph to show.

This table will work though:

| Date           | Data Usage |     | Cap  |     |
| -------------- | ----------:| --- | ---- | --- |
| 2021.03.01     |       0.04 | MB  | 2.05 | GB  |
| 2021.03.03     |       0.04 | MB  | 2.05 | GB  |
| 2021.03.17     |       9.15 | MB  | 2.05 | GB  |
| 2021.03.19     |       9.15 | MB  | 2.56 | GB  |
| 2021.03.24     |      98.88 | MB  | 2.50 | GB  |

FWIW, this is my tracker codeblock:

```tracker
searchType: table
searchTarget: __inbox/T-Mobile Data Tracking[0][0], __inbox/T-Mobile Data Tracking[0][1]
xDataset: 0
line:
    yAxisLocation: none, left
    lineColor: none, yellow
    showLegend: true
	fillGap: true
```

**Comment by pyrochlore** [comment]
You are right, currently it is not supported.
I will relax the date parsing for wiki links.

**Comment by pyrochlore** [comment]
Hi @DaveHenning, 

The enhancement was implemented and just released with the new version 1.6.0.
